From f88019960ad5319a6485bb890556fb0ba575a705 Mon Sep 17 00:00:00 2001
From: Anton Akhmerov <anton.akhmerov@gmail.com>
Date: Tue, 3 Jun 2025 13:40:46 +0200
Subject: [PATCH] disable scotch

---
 mumps/mumps.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/mumps/mumps.py b/mumps/mumps.py
index 4a43746..fd92253 100644
--- a/mumps/mumps.py
+++ b/mumps/mumps.py
@@ -71,6 +71,9 @@ def possible_orderings():
 
         possible_orderings.cached = ["auto"]
         for ordering in [0, 2, 3, 4, 5, 6]:
+            if ordering == 3:
+                # Fails on conda on osx-arm64
+                continue
             data = np.asfortranarray([1, 1], dtype=np.complex128)
             row = np.asfortranarray([1, 2], dtype=_mumps.int_dtype)
             col = np.asfortranarray([1, 2], dtype=_mumps.int_dtype)
-- 
2.49.0

