diff --git a/pyproject.toml b/pyproject.toml
index c141158..8a39131 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -27,7 +27,7 @@ classifiers = [
     "Operating System :: MacOS :: MacOS X",
     "Operating System :: Microsoft :: Windows"
 ]
-requires-python = ">= 3.12"
+requires-python = ">= 3.10"
 dependencies = [ "numpy>=2.0", "scipy>=1.13.0"]
 dynamic = ["version"]
 
@@ -64,7 +64,7 @@ MKL_DYNAMIC = "FALSE"
 MKL_NUM_THREADS = "1"
 
 [tool.pixi.dependencies]
-python = ">=3.12"
+python = ">=3.10"
 scipy = ">=1.13"
 numpy = ">=2.0"
 setuptools_scm = ">=8.3.1,<9"
@@ -88,7 +88,7 @@ license = "BSD-2-Clause"
 backend = { name = "pixi-build-python", version = "==0.4.0" }
 
 [tool.pixi.package.host-dependencies]
-python = ">=3.12"
+python = ">=3.10"
 cython = ">=3.1.1"
 meson = ">=1.8.0"
 meson-python = ">=0.18.0"
@@ -102,7 +102,7 @@ ninja = ">=1.12.1,<2"
 pkg-config = "*"
 
 [tool.pixi.package.run-dependencies]
-python = ">=3.12"
+python = ">=3.10"
 numpy = ">=2.0"
 scipy = ">=1.13"
 mumps-seq = ">=5.8.1,<6"
